### PR TITLE
build: use only gettext

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

**Comment by mbkma** [comment]
Building and installing works now for me :+1:

`make distcheck` is still failing. I think you'll need to remove

```bash
if test -z "$disabled_plugins"
then
       disabled_plugins="none"
fi
```

in `configure.ac` and to add a ChangeLog file.

**Comment by rbuj** [comment]
@mbkma done. Test:
```
$ ./autogen.sh --prefix=/usr && make && make distcheck
```

**Review by mbkma — APPROVED** [review]
Thanks! `make`, `make install`, `make uninstall` and `make distcheck` work now.

**Comment by mbkma** [comment]
Btw, this PR fixes https://github.com/mate-desktop/pluma/issues/206 and (partly) fixes https://github.com/mate-desktop/pluma/issues/168.

File: Makefile.am [hunk 1]
```diff
@@ -4,6 +4,7 @@ ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
 EXTRA_DIST =			\
 	autogen.sh		\
+	makepot			\
```
Comment (raveit65) [inline Makefile.am:7]: Do we really need that in tarball for consumers?
I think `makepot` spript is more a developer tool which we need to generate the resource for transifex.

Comment (raveit65) [reply Makefile.am:7]: If someone really need it than he can clone the git repo.

**Comment by raveit65** [comment]
Guys, do you get notifications from the repo?
I am still wondering why i don't get them.

File: Makefile.am (hunk 1 above)
Comment (rbuj) [reply Makefile.am]: done

**Comment by rbuj** [comment]
> Guys, do you get notifications from the repo?
> I am still wondering why i don't get them.

Gmail takes a while to receive the github email notifications. The continuous integration takes much longer than before, it should also be borne in mind that some projects build their dependencies from the sources, in addition we now carry out more analyzes. I usually check my mailbox daily, however I may take a few days to reply, I try to do it in less than 24 hours if it comes from a merge review, a week at most.

**Comment by raveit65** [comment]
Now, i received notification from your post immediately since I took part in this PR,
but i didn't received the opening notification from this PR and all others.
Normally, i get notification for all new reports and PRs.

**Comment by mbkma** [comment]
@raveit65 Maybe go to settings->notifications and check whether notifications are active for you. Also you can rename this repo to decouple it from my repository.

**Review by raveit65 — APPROVED** [review]
LGTM,
the terminal and codecomment moduls are working nicely.
No idea to test the syntex plugin.

**Comment by raveit65** [comment]
> Also you can rename this repo to decouple it from my repository.

I will do now.

**Pull request merged**